### PR TITLE
Jean

### PR DESCRIPTION
**Pull request by JeanFraga**

### CODE REVIEW

File: src/10_functions.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(num):
+    if num % 2 == 0:
```
Comment (jdorety) [inline src/10_functions.py:5]: replace `if` with `return`, and you've got a one-line function that does the same thing :smile_cat:

File: src/12_scopes.py [hunk 2]
```diff
@@ -16,11 +17,12 @@ def change_x():
 # This nested function has a similar problem.
 
 def outer():
+    global y
     y = 120
 
     def inner():
+        global y
```
Comment (jdorety) [inline src/12_scopes.py:24]: how could you achieve this without using the global keyword twice?  There's another keyword that you would only need to use once.

File: src/13_file_io.py [hunk 3]
```diff
@@ -11,9 +11,26 @@
...
+c = 'line three.'
+with open('src/bar.txt', 'w') as f:
+    f.write(a)
+    f.write(b)
+    f.write(c)
+f.closed
+print("\n")
+with open('src/bar.txt') as f:
+    read_data = f.read()
+print(read_data)
```
Comment (jdorety) [inline src/13_file_io.py]: Couldn't get this one to run, but could just be some cross-sytem complications

File: src/10_functions.py (hunk 1 above)
Comment (JeanFraga) [reply src/10_functions.py:5]: cool thank you

File: src/12_scopes.py (hunk 2 above)
Comment (JeanFraga) [reply src/12_scopes.py:24]: I couldn't find it :(

File: src/14_cal.py [hunk 4]
```diff
@@ -29,4 +29,41 @@
...
+# 2 args should set the month and year
+elif len(args) == 3:
+  month = int(args[1])
+  year = int(args[2])
+
+# Otherwise error and usage message
+else:
+  print("ERROR: Should be in format '14_cal.py [month] [year]'")
+  exit(0)
+
```
Comment (jdorety) [inline src/14_cal.py:61]: If no arguments are input, the program should return the current month/date.

**Review by jdorety — APPROVED** [review]
Good work overall.  Left a few notes on the files, but overall looks like you're getting the hang of it!

**Pull request merged**